identifier: CVE-2014-0472
title: Unexpected code execution using reverse()
description: "Django incorrectly handle dotted Python paths when using the django.core.urlresolvers.reverse
  function. An attacker can use this issue to cause Django to import arbitrary modules
  from the Python path, resulting\r\nin possible code execution."
date: "2014-04-21"
affected_range: '>=1.4,<1.4.11||>=1.5,<1.5.6||>=1.6,<1.6.3'
fixed_versions:
- 1.4.11
- 1.5.6
- 1.6.3
affected_versions: 'All versions starting from 1.4 before 1.4.11, all versions starting
  from 1.5 before 1.5.6, all versions starting from 1.6 before 1.6.3'
solution: Upgrade to latest
credit: Benjamin Bach
urls:
- https://www.djangoproject.com/weblog/2014/apr/21/security/
uuid: 1f9b27c0-d25e-48f2-b035-4cc691b0967d
package_slug: pypi/Django
cvss_v2: AV:N/AC:H/Au:N/C:P/I:P/A:P
